pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/LoadManager.java

```diff
@@ -63,7 +65,7 @@ public interface LoadManager {
     Optional<ResourceUnit> getLeastLoaded(ServiceUnitId su) throws Exception;
 
     default CompletableFuture<Optional<LookupResult>> findBrokerServiceUrl(
-            Optional<ServiceUnitId> topic, ServiceUnitId bundle) {
+            Optional<ServiceUnitId> topic, NamespaceBundle bundle, LookupOptions options) {
         throw new UnsupportedOperationException();
     }
```

Comment:
It's better to keep the API compatibility since it's an interface that can be configured. You can keep the 2nd parameter as `ServiceUnitId` and add a new default method.